tests/integration/run_integration_tests.sh

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+RED="\033[0;31m"
+GREEN="\033[0;32m"
+NC="\033[0m"
+SUCCESS="${GREEN}SUCCESS${NC}"
+FAILED="${RED}FAILED${NC}"
+SCRIPT_FAILED="${RED}SCRIPT EXITED UNEXPECTEDLY${NC}"
+
+DEFAULT_CTX=$(kubectl config current-context)
+
+source tests/integration/cleanup.sh
+
+IS_INTEGRATION_TEST_RUNNING="false"
+EXTENDED="false"
+
+cleanup() {
+    rv=$?
+    echo "Cleanup local resources..."
+    eval $(minikube docker-env --unset) >/dev/null 2>&1 || true
+    docker stop alerting-endpoint notary-signer notary-server >/dev/null  2>&1 || true
+    docker rm alerting-endpoint notary-signer notary-server >/dev/null 2>&1|| true
+    rm .hosts > /dev/null 2>&1 || true
+    rm -rf tmp >/dev/null 2>&1 || true
+    echo "Finished cleanup"
+    exit $rv
+}
+
+# cleanup on ^C
+cleanup_on_sigint() {
+    if [[ ${IS_INTEGRATION_TEST_RUNNING} == "true" ]]; then
+      preserve_info_and_cleanup
+    fi
+}
+
+trap "cleanup_on_sigint" SIGINT
+trap "cleanup" SIGTERM EXIT
+
+help()
+{
+   # Display Help
+   echo "This script will run the integration test suite locally. By default, all integration tests besides of the 'self-hosted-notary' test will be run (addable with -e (extended) flag)."
+   echo
+   echo "Flag   Values                       Meaning         Description"
+   echo "s      space-separated list         skip            Provided integration tests will be skipped. Cannot be used with -r (run-only) flag."
+   echo "       of integration test names"
+   echo "       as stings"
+   echo "r      space-separated list         run-only        Only the provided integrations tests will be run. Cannot be used with -s (skip) flag."
+   echo "       of integration test names"
+   echo "       as stings"
+   echo "e                                   extended        Include 'self-hosted-notary' test."
+   echo "c       <'minikube'|'kind'>         cluster         Specifies the cluster that should be used for integration testing. Must be either 'minikube' or 'kind'. (Default: 'minikube')"
+   echo "p       <path>                      preserve-logs   Must be a relative or absolute path. If set, log files of failed tests are copied that path."
```

Comment:
```suggestion
   echo "p       <path>                      preserve-logs   Must be a relative or absolute path. If set, log files of failed tests are copied to that path."
```

Reply:
Also "Must be a relative or absolute path." is somewhat tautological and could be "Must be a (valid?) path"